tailwind.config.js

```diff
@@ -84,13 +87,16 @@ module.exports = {
         h5: '1rem',
         h6: '0.875rem',
       },
-    },
-    screens: {
-      sm: '414px',
-      md: '834px',
-      lg: '874px',
-      xl: '946px',
-      '2xl': '1440px',
+      container: {
+        center: true,
+      },
+      spacing: {
+        4.5: '1.125rem',
+        15: '3.75rem',
+      },
+      height: {
+        58: '14.375rem',
+      },
```

Comment:
Could you use css module and remove this? IMHO, it's better not to bloat the config file.